### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by nategbrown9**

### CODE REVIEW

File: SwiftPackage/Binaries/BridgeClient.xcframework/ios-arm64/BridgeClient.framework/Info.plist [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <key>CFBundleExecutable</key>
     <string>BridgeClient</string>
     <key>CFBundleIdentifier</key>
-    <string>BridgeClient</string>
+    <string>org.sagebionetworks.bridge.kmm.shared.BridgeClient</string>
```
Comment (liujoshua) [inline SwiftPackage/Binaries/BridgeClient.xcframework/ios-arm64/BridgeClient.framework/Info.plist:8]: Is this right? Not sure why this changed

I see a commit say you updated xcframework, what did you do to update?

**Review by liujoshua — APPROVED** [review]
Looks good, thanks. One iOS-related question

File: SwiftPackage/Binaries/BridgeClient.xcframework/ios-arm64/BridgeClient.framework/Info.plist (hunk 1 above)
Comment (nategbrown9) [reply SwiftPackage/Binaries/BridgeClient.xcframework/ios-arm64/BridgeClient.framework/Info.plist:8]: I ran xcframework_build_debug.sh and checked in the results. Just realized I should have run xcframework_build_releash.sh, so I ran release version and push resulting artifacts.

I'm not sure why the CFBundleIdentifer changed, maybe @syoung-smallwisdom has an idea?

Comment (syoung-smallwisdom) [reply SwiftPackage/Binaries/BridgeClient.xcframework/ios-arm64/BridgeClient.framework/Info.plist:8]: Not sure.

**Review by syoung-smallwisdom — APPROVED** [review]
Looks good. Not sure why the id changed but it shouldn't matter.

**Pull request merged**